src/community/backup-restore/core/src/test/resources/applicationContext.xml

```diff
@@ -1,28 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
+
 <!-- <!DOCTYPE beans PUBLIC "-//SPRING//DTD BEAN//EN" "http://www.springframework.org/dtd/spring-beans.dtd"> -->
 <!-- 
  (c) 2017 Open Source Geospatial Foundation - all rights reserved
  This code is licensed under the GPL 2.0 license, available at the root
  application directory. 
 -->
-<beans xmlns="http://www.springframework.org/schema/beans"
-	xmlns:batch="http://www.springframework.org/schema/batch"
-	xmlns:util="http://www.springframework.org/schema/util"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="
-		http://www.springframework.org/schema/beans
-        http://www.springframework.org/schema/beans/spring-beans-4.0.xsd
-        http://www.springframework.org/schema/batch
-        http://www.springframework.org/schema/batch/spring-batch-3.0.xsd
-        http://www.springframework.org/schema/util
-        http://www.springframework.org/schema/util/spring-util-4.0.xsd">
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:batch="http://www.springframework.org/schema/batch" xmlns:util="http://www.springframework.org/schema/util" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="   http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-4.0.xsd         http://www.springframework.org/schema/batch         http://www.springframework.org/schema/batch/spring-batch-3.0.xsd         http://www.springframework.org/schema/util         http://www.springframework.org/schema/util/spring-util-4.0.xsd">
```

Comment:
One thing that is bothering me some, is that the elements are kept on a single line, despite the number of attributes they might contain. However, the plugin has no option for line wrapping. That said, I still prefer to output of the reforat to the current status quo :-D

Reply:
yes. 
I find the many extra whitespaces in the schemaLocation a bit odd, perhaps those can be fixed with a second run